sass/themes/schemas/components/light/_calendar.scss

```diff
@@ -9,1146 +9,2965 @@
-/// @prop {Map} date-selected-current-hover-background [color: ('secondary', 700)] - The hover background color for the selected/current date.
-/// @prop {Map} date-selected-current-focus-background [color: ('secondary', 700)] - The focus background color for the selected/current date.
-/// @prop {Map} current-outline [color: ('gray', 600)] - The outline color for the current date.
-/// @prop {Map} current-outline-hover [color: ('surface', 500)] - The outline color for the current date on hover.
-/// @prop {Map} current-outline-focus [color: ('surface', 500)] - The outline color for the current date on focus.
-/// @prop {Map} current-selected-outline [color: ('surface', 500)] - The outline color for the current selected date.
-/// @prop {Map} date-current-bg-color [color: ('surface', 500)] - The background color for the current date.
-/// @prop {Map} date-current-hover-background [color: ('secondary', 100)] - The hover background color for the current date.
-/// @prop {Map} date-current-focus-background [color: ('secondary', 100)] - The focus background color for the current date.
-/// @prop {Map} date-special-background [color: ('gray', 100)] - The background color used for special dates.
-/// @prop {Map} date-special-text-color [color: ('gray', 900)] - The text color used for special dates.
-/// @prop {Map} date-disabled-text-color [color: ('gray', 500)] - The text color for disabled dates.
-/// @prop {Map} date-disabled-background [color: ('secondary', 50)] - The background color for disabled dates.
-/// @prop {Map} week-number-color [color: ('gray', 500)] - The text color of the week number column.
+/// @prop {Map} content-foreground [color: ('gray', 900)] - The foreground color of the days, months and years views.
+/// @prop {Map} content-background [color: 'surface'] - The background color of the days, months and years views.
+/// @prop {Map} picker-background [color: 'surface'] - The background color of the month and year pickers.
+/// @prop {Map} picker-hover-foreground [color: 'secondary' 500] - The :hover foreground color of the month and year pickers.
+/// @prop {Map} picker-focus-foreground [color: 'secondary' 500] - The :focus foreground color of the month and year pickers.
+/// @prop {Map} weekday-color [color: ('gray', 500)] - The foreground color of the weekday labels.
+/// @prop {Map} weekend-color [color: ('gray', 900)] - The foreground color of the weekend days.
+/// @prop {Map} inactive-color [color: ('gray', 500)] - The foreground color of the previous and next month dates.
+/// @prop {Map} week-number-foreground [color: ('gray', 500)] - The foreground color of the week number column.
 /// @prop {Map} week-number-background [color: ('gray', 200)] - The background color of the week number column.
+/// @prop {Map} year-hover-foreground [contrast-color: ('gray', 200)] - The :hover foreground color of the year in years view.
```

Comment:
This value of year-hover-foreground is used nowhere and it's overridden in all themes by gray, 900. So you can just put gray-900 in the common schema and remove it from all other schemas.